code/modules/hydroponics/grown/citrus.dm

```diff
@@ -61,6 +73,18 @@
 	juice_results = list(/datum/reagent/consumable/orangejuice = 0)
 	distill_reagent = /datum/reagent/consumable/ethanol/triple_sec
 
+/obj/item/reagent_containers/food/snacks/grown/citrus/orange/attackby(obj/item/item, mob/user, params)
+	if(!istype(item ,/obj/item/kitchen/knife) && INTENT_HARM)
```

Comment:
what i said about the other ones applies here.
also youre copypasting this a lot. you probably want to make it a component, or why not just make it a part of the slicing system we have for like watermelons?

Reply:
i'll look into that when i add other garnishes, thanks